docs/fga-ownership-handoff.md

```
@@ -0,0 +1,369 @@
+    resourceId: id,
+    name,
+  })
+  for (const warning of result.warnings) {
+    mastra.getLogger().warn(`[FGA] ${warning}`)
+  }
+}
+```
+
+## The Warning You Saw
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)